### PR TITLE
Hello Microverse Project 

### PR DESCRIPTION
**Pull request by Srilearns**
Add Index.html and style.css to Hello-Microverse-Project Repository
Add Linters to Hello-Microverse-Project Repository
Add README.MD file to Hello-Microverse-Project Repository
Add Heading 'Hello Microverse'
Apply CSS styles to heading and background

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,9 +62,9 @@ For more information on the importance of a professional README for your reposit
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [your_project_name] <a name="Hello Microverse Project"></a>
```
Comment (franclobo) [inline README.md]: - [ ] Well done using the README template from Microverse, please modify this template according to the current project 👇🏽

- [ ] Replace  `your_project_name` with your project name 😂 

- Remove the guidelines and include only the relevant sections that apply to your project we are not using React Express or PostgreSQL in this project 👀 
image

- [ ] Please remove the LIVE DEMO SECTION or replace the link with a `Comming soon` message

- [ ] In the Getting Started section kindly describe how a new developer could make use of your project.

- [ ] In the Author section, replace Author1 with your name, provide the links to your social media, and remove Author2 unless you were doing a pair programming activity.
- [ ] Give credit to everyone who inspired your codebase. `I would like to thank... (Microverse could be a good option)`
- [ ] The FAQ section is optional it means whether there are no questions you can remove it.
- [ ] Please remove the note below the LICENSE section and make sure that your link is working.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Srilearns, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Descriptive branch name ✔️
- Linters installed ✔️ 
- Correct use of .gitignire file ✔️  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -124,19 +98,19 @@ For more information on the importance of a professional README for your reposit
 
 > Describe how a new developer could make use of your project.
 
-To get a local copy up and running, follow these steps.
+To get a local copy running,please follow these steps.
 
-### Prerequisites
+-npm
+```sh
```
Comment (Alusp) [inline README.md]: - [ ] Good job in implementing the changes requested by the previous reviewer, however, please remove this line of code as it is just meant to highlight specific areas in your code. :slightly_smiling_face:

File: README.md [hunk 3]
```diff
@@ -147,7 +121,7 @@ Example commands:
 
 ```sh
```
Comment (Alusp) [inline README.md]: `<!--`
`Example commands:`

````sh`

- [ ] Please remove the above line of code as it is just meant to highlight specific areas in your code. :slightly_smiling_face:

File: README.md [hunk 4]
```diff
@@ -147,7 +121,7 @@ Example commands:
...
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone git@github.com:Srilearns/Hello-Microverse-Project
 ```
 --->
```
Comment (Alusp) [inline README.md]: `--->`

- [ ] Please remove the above line of code as it is just meant to highlight specific areas in your code. 🙂

File: README.md [hunk 5]
```diff
@@ -208,75 +182,19 @@ Example:
 
 > Mention all of the collaborators of this project.
```
Comment (Alusp) [inline README.md]: `> Mention all of the collaborators of this project.`

- [ ] Please remove the above line of code as it is just meant to tell you what to do in your code. slightly_smiling_face

File: README.md [hunk 6]
```diff
@@ -286,6 +204,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (Alusp) [inline README.md]: - [ ] Please ensure that you resolve your `License` as link is broken, see below screenshot: :arrow_down: 

Student 404

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @Srilearns , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descrptive ✔️
- [ ] Added .html and .css files ✔️
- [ ] node_modules properly ignored ✔️
- [ ] Test.md file ignored successfully ✔️


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] [OPTIONAL] PLease always remember to have a descriptive branch name, using descriptive branch names promotes transparency, organization, and effective collaboration in a development workflow. It enhances productivity and reduces confusion, making it an essential best practice in version control systems like Git. :slightly_smiling_face: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Srilearns, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Fair README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**